src/desktop_notifier/macos_support.py

```diff
@@ -71,4 +75,12 @@ def is_signed_bundle() -> bool:
         None,
     )
 
-    return cast(int, signed_status) == 0
+    if cast(int, signed_status) == 0:
+        return True
+    else:
+        _log_unsigned_warning(f"signed_status is {signed_status}")
```

Comment:
And "Cannot verify app signature. SecStaticCodeCheckValidity() failed with OSStatus: %s"?